packages/styles/variables.import.styl

```diff
@@ -79,5 +79,5 @@ $min-font           = 12px
 $max-font           = 20px
 $min-width          = 768px
 $max-width          = 1200px
-$vertical-rhythm    = true
+$vertical-rhythm    = false
```

Comment:
http://webtypography.net/2.2.2
It sets a margin on headers which conflicts with other styles.